Module/uts/custom002.uts.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
Another resref situation here - "custom002" doesn't describe it well. Maybe something like "imp_droid_snds" would work.